src/layouts/HolidayLayout/_components/HolidayDiscountBundle.astro

```diff
@@ -9,20 +9,24 @@ interface Props {
 }
 
 const { bundleImage, offer, prices, title } = Astro.props;
+const productId = 5951531;
+const checkoutURL = `https://courses.rockthejvm.com/purchase?product_id=${productId}`;
 ---
 
 <section class="mb-8 min-h-[60vh] md:mb-16">
   <div class="relative isolate">
     <div class="mx-auto max-w-7xl sm:px-6 lg:px-8">
       <div
-        class="mx-auto flex max-w-2xl flex-col gap-16 px-6 sm:rounded-3xl sm:p-8 lg:mx-0 lg:max-w-none lg:flex-row lg:items-center xl:gap-x-20 xl:px-20"
+        class="mx-auto flex max-w-2xl flex-col gap-16 px-6 sm:p-8 lg:mx-0 lg:max-w-none lg:flex-row lg:items-center xl:gap-x-20 xl:px-20"
       >
-        <Picture
-          src={bundleImage}
-          alt={`${title} bundle`}
-          class="w-full flex-none lg:max-w-lg"
-          formats={["avif", "webp"]}
-        />
+        <a href={checkoutURL}>
```

Comment:
As a user my expectation for clicking on the image would be to take me to the "learn more" page.

Reply:
Done